TfIdf_BOW_creator.py

```diff
@@ -0,0 +1,186 @@
+import csv
+import numpy as np
+from regex_extraction import word_processor
+import re 
+import enchant
+dict = enchant.Dict("en_US")
+from nltk.stem import PorterStemmer
+ps = PorterStemmer()
+
+# this part generate tf-idf BOW
+
+def sentence_processor(sentence):
+    processed_sentence  = ""
+    list_string = sentence.split(' ')
+    for word in list_string:
+        word = word.lower()
+        word = re.sub(r'[^a-z\s]','',word)
+        word = ps.stem(word)
+        if len(word)>2 and dict.check(word):
+            processed_word = word
+        else:
+            processed_word = ""
+
+        processed_sentence+=" "+processed_word
+    return processed_sentence
+
+# Create vocab
+def create_vocab(filename):
+    # open the file in read mode
+    read_file = open(filename, 'r')
+    
+    # creating dictreader object
+    file = csv.DictReader(read_file)
```

Comment:
All these functionality probably could be simplified using the pandas library ...